firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -73,8 +73,8 @@ class SearchViewController: SiteTableViewController,
     private let viewModel: SearchViewModel
     private let model: SearchEngines
     private var suggestClient: SearchSuggestClient?
-    private var remoteClientTabs = [ClientTabsSearchWrapper]()
-    private var filteredRemoteClientTabs = [ClientTabsSearchWrapper]()
```

Comment:
nit - seems like we're granting internal access for the sake of tests, do we want to keep access level private and use `profile.mockClientAndTabs` to set our tabs and check `data` instead of opening up `filteredRemoteClientTabs`?